### DISCUSSION TITLE
Is it possible to stop the load test during mid of the test duration and get the report

### DISCUSSION THREAD

**Discussion by rrnareshkumar** [artilleryio/artillery#2110] [Artillery]
Hi , I designed load test scenario to run for 2 hours ,generating load from my local windows machine. In case If i had to stop the test in the mid (ex: after 1 hour) , Is it possible to stop the load test and get the report with metrics for initial 1 hour duration ?

**Comment by bernardobridge** [comment]
Hi @rrnareshkumar 👋 . 

No, currently that is not possible as the final report doesn't get generated, but we'll keep that in mind as a feature request. Can you explain a little bit more what is your use case for this?

_Note: in our [Artillery Cloud Dashboard](https://github.com/artilleryio/artillery/discussions/1948) which is currently in beta, you will be able to see metrics up to the point of a test being cancelled. While the final (aggregated) report still won't get generated, this will still be a useful way to see how the test was doing by analysing the intermediate reports until the point of cancellation._
